crates/common_utils/src/crypto.rs

```diff
@@ -1,10 +1,13 @@
 //! Utilities for cryptographic algorithms
-use data_encoding::BASE64;
+use base64::Engine;
```

Comment:
no we already have the same in /crates/router/src/consts.rs we don't need this new const. use the existing one